### ISSUE TITLE
RtpPacketizationConfig not properly initialized

### ISSUE THREAD

**Issue by fusionfoxy** [paullouisageneau/libdatachannel#1216]
The RtpPacketizer takes a RtpPacketizationConfig when created. After creation, this config must have the `playoutDelayId` initialized manually eg.:
```
        auto rtpConf_ = std::make_shared<rtc::RtpPacketizationConfig>(42, "foo", 96, 90000);
        rtpConf_->playoutDelayId = 0;
```
Otherwise, valgrind will complain about a `Conditional jump or move depends on uninitialised value` at
```
==48363==    at 0x312384: rtc::RtpPacketizer::packetize(std::shared_ptr<std::vector<std::byte, std::allocator<std::byte> > >, bool) (rtppacketizer.cpp:34)
```
I would expect the config class to initialize this value automatically.

**Comment by paullouisageneau** [comment]
Related to https://github.com/paullouisageneau/libdatachannel/pull/1152/files

I'll push a fix.

**Comment by paullouisageneau** [comment]
Thank you for reporting, it is fixed by https://github.com/paullouisageneau/libdatachannel/pull/1217. The fix will be released in v0.21.2.
